context/Auth.js

```diff
@@ -0,0 +1,79 @@
+import { useRouter, useSegments } from "expo-router";
```

Comment:
Maybe add a variable that will automatically log people in as John Doe for testing?

Reply:
I added a flag for this via `yarn test` (optional arguments: `--ios`, `--android`, `--tunnel`)

References
- https://docs.expo.dev/guides/environment-variables/
- https://www.npmjs.com/package/cross-env (used to avoid different syntax for setting `NODE_ENV` on Unix vs Windows)